src/metadata/metadata_raw_dynamic.c

```diff
@@ -383,7 +383,7 @@ static int raw_dynamic_load_all_update(struct ocf_request *req)
 
 	for (i_page = 0; i_page < count; i_page++, context->i++) {
 		if (!context->page) {
-			context->page = env_malloc(PAGE_SIZE, ENV_MEM_NORMAL);
+			context->page = env_secure_alloc(PAGE_SIZE);
```

Comment:
There's env_free() called in raw_dynamic_load_all_complete() for this allocation. Should be secure version.